cmd/manager/main.go

```diff
@@ -30,7 +30,16 @@ import (
 )
 
 func main() {
+	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
+	klog.InitFlags(klogFlags)
 	flag.Parse()
+	flag.VisitAll(func(f1 *flag.Flag) {
```

Comment:
@vincepri - your change in the aws provider (and my corresponding change in the gcp provider) don't try to synchronize flags between the built in flags and klog flags. Should we? If this is necessary, shouldn't the klog libraries just handle this transparently so that this code block isn't required in every main function?

Reply:
I think it’s actually what it does, we want to sync the flags between klog and glog if they coexist. I agree that it might be better to have this ready in klog, that said it’s a change that we should bring up in klog itself.

Reply:
[This](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/cmd/manager/main.go#L37-L46) is what we do in the aws provider for context.